### PR TITLE
chore: [P4PU-375] added test for user info

### PR DESCRIPTION
**Pull request by oleksiybozhykntt**
#### List of Changes
This PR include postman test that are required to demonstrate the user info functionality

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: postman/pagopa-arc-E2E.postman_collection.json [hunk 1]
```diff
@@ -2608,6 +2608,66 @@
...
+                  "protocolProfileBehavior": {
+                    "followRedirects": true
+                  },
+                  "request": {
+                    "auth": {
+                      "type": "bearer",
+                      "bearer": [
+                        {
+                          "key": "token",
+                          "value": "eyJhbGciOiJSUzUxMiIsInR5cCI6IkpXVCJ9.eyJ0eXAiOiJCZWFyZXIiLCJpc3MiOiJhcHBsaWNhdGlvbi1hdWRpZW5jZSIsImp0aSI6ImFhZmUzMGRiLTVlNDktNDc2ZS04YzhjLWU3OWJmNDk5YzJlNiIsImlhdCI6MTcyMzIxNzM4NCwiZXhwIjoxNzIzMjIwOTg0fQ.LFI0YFjS7bjv3VYGDTR3GgdPFbbUKrza2H8eeh7g6OtzBhcor0U0B_M_8LI98s-eob3TpWM9wq8zTwnvZtz_tmI37FU83RJF513Mrhi5YBYR8MI78XMSmRH7xRPIzlIAruCYbcv6JVs5daf7UKLjWGZWr3cXH7-TzWoA2OAE2MhzzI23R8b8pXDXrRearR5blyhEM1Wk4TRz3RZSksajGRQQ_gwO0K_8pT3zpk-hsZfWZ2iGwjWdKsTBwSmhYL68Qns2ic5McvM_gC5_xW3hrzqTP27NQJnkRjdT03uIFXkZSrLRBbgzPs286k4DiFaWMrASWofv35AQvqalUIXwmQ",
```
Comment (and-mora) [inline postman/pagopa-arc-E2E.postman_collection.json:2650]: where did you get this token from?

Comment (oleksiybozhykntt) [reply postman/pagopa-arc-E2E.postman_collection.json:2650]: it is a sample application token used once for testing

Comment (and-mora) [reply postman/pagopa-arc-E2E.postman_collection.json:2650]: the test consists in calling the endpoint /user and get 200 as response with any token. This behaviour will broke when the token validation will be activated, correct?

Comment (oleksiybozhykntt) [reply postman/pagopa-arc-E2E.postman_collection.json:2650]: yes, this test will fail, also this test should be improved once the stategy for token generation will be defined

**Review by and-mora — APPROVED** [review]

**Pull request merged**